client/app/hosting/hosting.service.js

```diff
@@ -170,6 +172,17 @@
                 }).then((hosting) => {
                     hosting.isCloudWeb = _(hosting.offer).includes("CLOUD");
 
+                    if (hosting.isCloudWeb) {
+                        hosting.configurationQuota = this.HOSTING.cloudWeb.configurationQuota;
+                        hosting.totalQuota = _.clone(this.HOSTING.cloudWeb.configurationQuota);
+
+                        const configurationOctet = this.ConverterService.convertToOctet(hosting.configurationQuota.value, hosting.configurationQuota.unit);
+                        const quotaSizeOctet = this.ConverterService.convertToOctet(hosting.quotaSize.value, hosting.quotaSize.unit);
```

Comment:
**octet** (plural octets)
A group of eight musicians performing together. 

:notes:

Reply:
this.ConverterService.convertToOctet => octet